### PR TITLE
ENT-5282 Fix release numbers

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @craigcomstock can review this?

**Comment by Lex-2008** [comment]
latest build: https://ci.cfengine.com/job/pr-pipeline/4476/
packages: http://buildcache.cloud.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4476/

File: build-scripts/package [hunk 1]
```diff
@@ -160,10 +160,10 @@ case "$PACKAGING" in
     then
         # Create AIX bff packages
         chmod +x "$P/$PKG.bff.sh"
-        "$P/$PKG.bff.sh" "$RPM_VERSION.$safe_prefix$RPM_RELEASE" "$BASEDIR" "$PREFIX"
+        "$P/$PKG.bff.sh" "$RPM_VERSION-$safe_prefix$RPM_RELEASE" "$BASEDIR" "$PREFIX"
```
Comment (nickanderson) [inline build-scripts/package]: ? I don't see BFF listed in ENT-5282 as having an issue

**Comment by Lex-2008** [comment]
comment added to the ticket

**Pull request merged**